src/lib/flow/include/sol-flow-metatype.h

```diff
@@ -95,12 +95,12 @@ struct sol_flow_metatype_context {
      * @param ctx The metatype context.
      * @param name The file path.
      * @param buf Where the file contents should be stored.
```

Comment:
Explicitly say that the buffer must be already initialized and data will be appended. If the data does not fit, an error is returned